core/comment.js

```diff
@@ -10,43 +10,43 @@
  */
 'use strict';
 
-goog.provide('Blockly.Comment');
-
-goog.require('Blockly.browserEvents');
-goog.require('Blockly.Bubble');
-goog.require('Blockly.Css');
-goog.require('Blockly.Events');
+goog.module('Blockly.Comment');
+goog.module.declareLegacyNamespace();
+
+const Block = goog.requireType('Blockly.Block');
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+const Bubble = goog.require('Blockly.Bubble');
+const Coordinate = goog.requireType('Blockly.utils.Coordinate');
+const Events = goog.require('Blockly.Events');
+const Icon = goog.require('Blockly.Icon');
+const Size = goog.requireType('Blockly.utils.Size');
+const Svg = goog.require('Blockly.utils.Svg');
+const WorkspaceSvg = goog.requireType('Blockly.WorkspaceSvg');
+const {conditionalBind, Data, unbind} = goog.require('Blockly.browserEvents');
+const {createSvgElement, HTML_NS} = goog.require('Blockly.utils.dom');
+const {inherits} = goog.require('Blockly.utils.object');
+const {register} = goog.require('Blockly.Css');
+const {IE} = goog.require('Blockly.utils.userAgent');
```

Comment:
The most minor of nits, `const {IE} = ...` should be moved up so it is alphabetized.
 I THINK this should be before inherits, or at least before register. I'm not how sure how capitalization affects alphabetization.

Reply:
Yeah, the styleguide is sort of vague on that point since it assumes they get auto-sorted. I think capital/lowercase are sorted separately, but I realize the capitals should probably come first; any objections to that (so {IE} before {conditionalBind...})?

Reply:
I had this same import in a file I was working on and Chris had me require the entire module. [Comment found here](https://github.com/google/blockly/pull/5076#discussion_r671414763).

Reply:
Since the consensus seems to be requiring the module (which seems a bit more readable to me as well) I went ahead and did that, which also resolves the sort issue.